### DISCUSSION TITLE
1.0.6 Release question

### DISCUSSION THREAD

**Discussion by ThePotatoProgrammer** [majusko/pulsar-java-spring-boot-starter#148] [General]
Do you know when the latest 1.0.6 release will be available in the artifactory? Corporate red tape prevents our team from pulling directly from GitHub. Thanks.

**Comment by majusko** [comment]
Hi, @DigitalPotatoBits - 1.0.6 was already released containing TTL configuration - https://search.maven.org/artifact/io.github.majusko/pulsar-java-spring-boot-starter
